actiontext/lib/action_text/engine.rb

```diff
@@ -87,7 +87,9 @@ def to_trix_content_attachment_partial_path
 
     config.after_initialize do |app|
       if klass = app.config.action_text.sanitizer_vendor
-        ActionText::ContentHelper.sanitizer = klass.safe_list_sanitizer.new
+        ActiveSupport.on_load(:action_view_base) do
```

Comment:
i've opted for this hook because the `ContentHelper` is defined inside `app/helpers` as part of the engine.